contents/blog/clickhouse-announcement.md

```diff
@@ -0,0 +1,31 @@
+---
+date: 2021-08-13
+title: Why we’re moving you to ClickHouse - Benefits and tradeoffs
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: mo-shehu
+featuredImage: ../images/blog/filename.png
+featuredImageType: full
+---
+We're pleased to introduce ClickHouse as the underlying database for our event analytics. 
+
+[insert any other context here]
+
+## Why ClickHouse?
+
+ClickHouse is a database designed to enable fast, efficient analytics computations across multiple machines. We are moving [PostHog Open Source]() from PostgreSQL to ClickHouse because ClickHouse is a better overall experience on every level. Postgres runs out of capacity quickly and doesn’t support the advanced analytics we need for our product, and there is no way to scale PostgreSQL beyond a certain point.
+There are several benefits to using ClickHouse over PostgreSQL, such as:
+* Speed: ClickHouse is orders of magnitude faster than most other databases, mainly because of how tightly it couples storage and computing power, resulting in significantly quicker query response times.
+* Cost: With more efficient use of processing power, memory, and storage (plus horizontal scaling), you can use cheaper commodity boxes versus having to scale up using specialized (read: costlier) software.
+* Migrations: Moving over from another database? ClickHouse lets you create materialized versions of your data thanks to its advanced engine, resulting in exponentially faster migrations between deployments.
```

Comment:
we might also want to say that one can more easily migrate between posthog cloud and self hosted as both are backed by ClickHouse now

Reply:
I don't think the migration capabilities (a migration tool) are really there yet, but this does make us building them more possible, so I'd rather go for "will be able to" instead of "can" here 😅

Reply:
'Will be able to' I think is quite vague and potentially annoying for engineers though? (I am not an engineer obviously, but I would rather know it is possible, it isn't possible, or it isn't possible but it will be at X time)

Reply:
Tweaked the wording